analysis/R/read_input.R

```diff
@@ -40,18 +40,8 @@ ReadParameterFile <- function(params_file) {
   params
 }
 
-# Handle the case of redundant cohorts, i.e. the counts file needs to be
-# further aggregated to obtain counts for the number of cohorts specified in
-# the params file.
-#
-# NOTE: Why is this happening?
-AdjustCounts <- function(counts, params) {
-  apply(counts, 2, function(x) {
-    tapply(x, rep(1:params$m, nrow(counts) / params$m), sum)
-  })
-}
+ReadCountsFile <- function(counts_file, params) {
```

Comment:
It is called.  Please rebase all your commits as almost all of them have already been committed and are misleading in the change history.